app/models/framework/definition/RM3756.rb

```diff
@@ -84,7 +84,7 @@ class Invoice < EntryData
         field 'Customer Invoice Date', :string, exports_to: 'InvoiceDate', ingested_date: true
         field 'Customer Invoice Number', :string, exports_to: 'InvoiceNumber', presence: true
         field 'Service Type', :string, exports_to: 'ProductGroup', presence: true, case_insensitive_inclusion: { in: SERVICE_TYPE_VALUES }
-        field 'Primary Specialism', :string, exports_to: 'ProductDescription', presence: true, dependent_field_inclusion: { parent: 'Service Type', in: MAPPING }, case_insensitive_inclusion: { in: PRIMARY_SPECIALISM_VALUES, message: 'must match the selected service type. For a list of service types and specialisms, check the lookups tab in the template.' }
+        field 'Primary Specialism', :string, exports_to: 'ProductDescription', presence: true, dependent_field_inclusion: { parent: 'Service Type', in: MAPPING }
```

Comment:
If you're feeling like "Clean ALL the things!" [1], the constant `PRIMARY_SPECIALISM_VALUES` is now only used for the `mixture` mapping, and could be removed and replaced with summing up the two arrays in place.

[1] http://hyperboleandahalf.blogspot.com/2010/06/this-is-why-ill-never-be-adult.html

Reply:
@CristinaRO It's likely that in the act of replacing frameworks with FDL, this will have to be taken care of in the framework definition language transpiler, so I'm hoping we'll get away without having to do this (but having to remember that we will need to implement this in said transpiler, it's not in the spike's lookup validation [implementation](https://github.com/rgarner/parslet-spike/blob/133f9a5f9dee9f04f75093bd9b5cc926cdc2de34/lib/framework/definition/transpiler.rb#L66-L95))